R/dist.R

```diff
@@ -811,8 +852,13 @@ tune_inv_gamma <- function(lower = 2, upper = 21) {
 #' @param max Numeric, maximum value of the distribution. The distribution will
 #' be truncated at this value.
 #'
+#' @param pmf Numeric, defaults to an empty vector corresponding to a parametric
```

Comment:
This doesn't mention it being non-parametric etc.

Reply:
Resolved in https://github.com/epiforecasts/EpiNow2/pull/363/commits/a3815887b399e1a72d886a3f7a590ccb99dcb22f